src/Symfony/Component/Dsn/README.md

```diff
@@ -0,0 +1,13 @@
+Dsn Component
```

Comment:
```diff
- Dsn Component
+ DSN Component
```

Reply:
The component's name is really `Dsn Component`. See @javiereguiluz comment